innsender/src/main/resources/application.yml

```diff
@@ -127,6 +128,7 @@ spring:
     url: jdbc:postgresql://${DATABASE_HOST:localhost}:${DATABASE_PORT:5432}/${DATABASE_NAME:postgres}?user=${DATABASE_USERNAME:postgres}&password=${DATABASE_PASSWORD:postgres}
 
 status_log_url: "https://logs.adeo.no"
+fil-konvertering_url: ${KONVERTERING_TIL_PDF_URL}
```

Comment:
Flytt til toppen (for alle profiler) med fallback-verdi "http://localhost:3000"?